src/util/expr.h

```diff
@@ -52,14 +61,14 @@ class exprt:public irept
     add(ID_type, _type);
   }
 
-  // returns the type of the expression
+  /// Returns the type of the expression
```

Comment:
:pick: You just asked me to replace `returns` with `return` everywhere for codet. :wink:

Reply:
:flushed:

Reply:
Fixed